cmd/wallet-web/src/router/index.js

```diff
@@ -60,6 +60,7 @@ router.beforeEach(async (to, from) => {
         // later on, if we need to authenticate the same user again we just call requestAccess
         // if needed, it will return us a new continue access token to complete user authentication
         store.dispatch('updateGnapReqAccessResp', null);
+        store.dispatch('updateUserLoaded', true);
```

Comment:
@HeidiHan0000, you also need to dispatch in the end of every other guard, otherwise user won't be able to get past the loading spinner when signing back in, etc.